api/src/main/java/org/infinispan/api/common/CacheEntryVersion.java

```diff
@@ -0,0 +1,7 @@
+package org.infinispan.api.common;
+
+/**
+ * @since 14.0
+ **/
+public interface CacheEntryVersion {
```

Comment:
Should this be Comparable ? or is equality/inequality sufficient ?

Reply:
currently, only `equals()` is used.

Reply:
Ok